### DISCUSSION TITLE
Need "save the workspace/project" feature

### DISCUSSION THREAD

**Discussion by CarreraPHP** [zed-industries/zed#7194] [Feature Requests]
I am been using Zed for a week now and it's been a wonderful experience till date, initially i use to open each node project in one zed IDE instance but then i was not able to toggle between the windows using "CMD + tab". We can use "CMD + `" to toggle between the windows of the same application but that for some reason isn't comfortable or i always tend to press tab instead of tilde.

Then i realized that the "add for project" option is available in the menu even after i added a folder from the project panel and when selected, i was able to add other relevant projects to the one i am working on and continue in one single window.

This feature is referred as "Workspace" in VSCode and in VSCode they provide option to save the workspace which is missing in Zed.

I understand that if i close and reopen zed again the last working folders will be reopened. but, when i want to work on a different requirement and i close the current window and open a new folder in a new window. the workspace setup i created by adding multiple folder will be lost and i have to construct it again.

Please add the "save project" feature that would work similar to VSCode workspace feature.

**Comment by danieltsoftware** [comment]
Yup. Just like how it works in Sublime. I NEEEED THIS haha

**Reply by mloskot** [reply]
Vote for https://github.com/zed-industries/zed/issues/9459, I guess

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
